### PR TITLE
Added suru overlay to group pages

### LINKED ISSUE #130: Create suru overlay on group banners
#111111 and 65% opacity

    background: rgba(17, 17, 17, 0.85);

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Added suru overlay to group pages
- Small loses the overlay shape

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [group page](http://0.0.0.0:8023/cloud-and-server/)

## Issue / Card

Fixes #130

Signed-off-by: Peter Mahnke <peter@transitionelement.com>

### CODE REVIEW

File: static/sass/_pattern_strips.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@mixin insights-p-strips {
+  @include insights-p-strips-suru;
+}
+
+@mixin insights-p-strips-suru {
+  .suru {
```
Comment (anthonydillon) [inline static/sass/_pattern_strips.scss]: Please change to `.p-strip--suru` please

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Comment in the code

QA issue:
The drop downs dont work over the new cards

**Comment by spencerbygraves** [comment]
👍

**Review by anthonydillon — APPROVED** [review]
LGTM 👍

**Comment by frankiegranato** [comment]
I can't see the demo. I see a error page: IndexError: list index out of range :-(

**Comment by pmahnke** [comment]
@frankiegranato try again

**Comment by frankiegranato** [comment]
The suru fold thingy looks lovely guys.

**Pull request merged**